ClientCore/LoadingScreenController.cs

```diff
@@ -5,18 +5,19 @@ namespace ClientCore
 {
     public static class LoadingScreenController
     {
+        private static readonly Random _random = new Random();
```

Comment:
See https://github.com/CnCNet/xna-cncnet-client/pull/728 for the most preferred way to get the random object. We don't create a new one anymore